src/Vocup/Models/VocabularyBookStatistics.cs

```diff
@@ -76,4 +76,14 @@ public void Refresh()
 
         OnPropertyChanged();
     }
+
+    public void Reset()
```

Comment:
I don't know why, but I never designed this class for unloading a vocabulary book. I would like to challenge you to change its design, e.g. making `VocabularyBookStatistics` nullable in `MainForm.cs` or making `book` nullable in this class so that there a proper mechanism for loading and unloading statistics. When no book is loaded you could also disable the statics panel like it is already done for most of the buttons.

Reply:
I took a more detailed look at the data model. I did not do a very great job when I designed it 6 years ago. `VocabularyBookStatistics` is the wrong place to fix this bug, though. In my opinion it would make most sense to reset the counters in `MainForm.UnloadBook` whenever a full unload is performed.